chainntnfs/interface_test.go

```diff
@@ -107,7 +107,7 @@ func waitForMempoolTx(r *rpctest.Harness, txid *chainhash.Hash) error {
 }
 
 func testSingleConfirmationNotification(miner *rpctest.Harness,
-	notifier chainntnfs.ChainNotifier, t *testing.T) {
+	notifier chainntnfs.TestChainNotifier, t *testing.T) {
```

Comment:
Awesome split of commits, makes it very easy to see what changes in every step 🎉